README.md

```diff
@@ -1,2 +1,32 @@
 # django-prices-vatlayer
-Vatlayer API support for django-prices
+[Vatlayer API](https://vatlayer.com/) support for django-prices
+
+```python
+from prices import Price
+from django_prices_vatlayer.european_vat import get_price_with_vat
+
+price_with_vat = get_price_with_vat('SK', Price(10, currency='USD'), 'books')
+print(price_with_vat)
+# Price(net='10', gross='11', currency='USD')
+print(price_with_vat.history)
+# (Price('10', currency='USD') | EuropeanVat(SK, rate_name=books, vat_rate=10))
+```
+
+Installation
+==============
+First install the package:
+```
+pip install django-prices-vatlayer
+```
+Then add `'django_prices_vatlayer'` to your `INSTALLED_APPS`.
+
+Set following settings in your project's settings:
+
+ * `VATLAYER_ACCESS_KEY`
+ *  To avoid possible MITM attacks, Paid Customer might force secure connection by overriding `VATLAYER_API` endpoint
```

Comment:
How about we make it HTTPS by default when `DEBUG` is off and then state something like:

> To safeguard against DNS poisoning and man-in-the-middle attacks we recommend that `VATLAYER_API` is set to use the HTTPS endpoint which is the default when running in production mode. The secure endpoint is only available with paid plans.